db/seed.sql

```diff
@@ -26,6 +26,11 @@ INSERT INTO adoptable_creatures(
     type_id
   )
 VALUES
+<<<<<<< HEAD
```

Comment:
please resolve merge conflicts and remove merge markers prior to submitting PRs.